gnovm/stdlibs/std/emit_event.go

```diff
@@ -46,12 +59,20 @@ func attrKeysAndValues(attrs []string) ([]gnoEventAttribute, error) {
 }
 
 type gnoEvent struct {
-	Type       string              `json:"type"`
+	OrigCaller string              `json:"orig_caller"`
```

Comment:
We're thinking of removing OrigCaller to begin with

Reply:
if we're removing origin caller, what do you think of alternative way for getting it from event ??

Reply:
Just use PrevRealm.

OrigCaller is who originated the transation. In the context of a transaction result, or indexer, this information already exists, as the event is associated with the transaction itself.

Reply:
08fc1ff8f155ae874eb5cf2b80656685ebbf992e